regression/goto-instrument/nondet-static-exclude_subfolder/package1/file.c

```diff
@@ -0,0 +1,10 @@
+#include "assert.h"
+#include "stdint.h"
```

Comment:
`#include <assert.h>`, `#include <stdint.h>` unless you really want to have the option to use a local copy of those files?